kinode/src/main.rs

```diff
@@ -431,7 +431,7 @@ async fn main() {
      */
     let networking_keypair_arc = Arc::new(decoded_keyfile.networking_keypair);
 
-    let (kernel_process_map, db) = state::load_state(
+    let (kernel_process_map, db, reverse_cap_index) = state::load_state(
```

Comment:
solution to this:

since the reverse_cap_index contains zero new data, and is just a re-representation of data in process_map, don't persist it to disk. instead, re-populate the reverse-index on load.

Reply:
yeah that's what I'm doing right?